database/nameConstraints.js

```diff
@@ -0,0 +1,32 @@
+'use strict'
```

Comment:
i am of the opinion that we should no add these for the Udaru users, they will make their own constraints if they really want but probably that will cause issues with keeping the migration up to date.

Reply:
yes if we change constraints in future migrations theirs will be dropped...